test/controllers/EntryDetailsControllerSpec.scala

```diff
@@ -79,31 +90,88 @@ class EntryDetailsControllerSpec extends ControllerSpecBase {
     "payload contains valid data" should {
 
       "return a SEE OTHER response and redirect to correct location for date BEFORE EU exit" in new Test {
-        private val request = fakeRequest.withFormUrlEncodedBody(buildForm(day=Some("31"),month=Some("12"),year=Some("2020")):_*)
+        private val request = fakeRequest.withFormUrlEncodedBody(buildForm(day = Some("31"), month = Some("12"), year = Some("2020")): _*)
         lazy val result: Future[Result] = controller.onSubmit(request)
         status(result) mustBe Status.SEE_OTHER
         redirectLocation(result) mustBe Some(controllers.routes.AcceptanceDateController.onLoad().url)
       }
 
       "return a SEE OTHER response and redirect to correct location for date AFTER EU exit" in new Test {
-        private val request = fakeRequest.withFormUrlEncodedBody(buildForm(day=Some("02"),month=Some("01"),year=Some("2021")):_*)
+        private val request = fakeRequest.withFormUrlEncodedBody(buildForm(day = Some("02"), month = Some("01"), year = Some("2021")): _*)
         lazy val result: Future[Result] = controller.onSubmit(request)
         status(result) mustBe Status.SEE_OTHER
         redirectLocation(result) mustBe Some(controllers.routes.AcceptanceDateController.onLoad().url)
       }
 
       "update the UserAnswers in session" in new Test {
-        private val request = fakeRequest.withFormUrlEncodedBody(buildForm():_*)
+        private val request = fakeRequest.withFormUrlEncodedBody(buildForm(): _*)
         await(controller.onSubmit(request))
         verifyCalls()
       }
     }
 
+    "not in change mode after successful submission" should {
+      "not in change mode after successful submission redirect to Acceptance date page" in new Test {
```

Comment:
this is a repeat of one of the tests in "payload contains valid data". Suggest deleting the repeated test and moving the other on into the valid payload section.